dangerous_test_thirdparty_version.sh

```diff
@@ -47,102 +42,15 @@ test_dir=$dir/test/shell
-    "fa01b414674cb38adc90ccf7a2042e82198dbb19dc41faccf0b5941ec08b1998" \
-    "e435d5ef31cc12dbf66719b7d5ab677ad739c63c3e451757b9688dcbeda0a984" \
-    "d932f809012d2cf832226b52a8bd82ed35b0257b1471c98968c0cd9ddf5327ab"
-
-# Intermediate versions of 2.12.x
-
-$runner test_scala_version "2.12.1" \
-    "fdd7679ce8a3fb4e30fbb9eaf9451f42c042f5ac3b5497f0fd01c379a3df9b3f" \
-    "9dab78f3f205a038f48183b2391f8a593235f794d8129a479e06af3e6bc50ef8" \
-    "d8a2b9d6d78c7457a40e394dc0c4fa6d6244acf0d156bbbcb311a9d497b85eec"
-
-
-$runner test_scala_version "2.12.2" \
-    "b3d41a2887c114821878d45c1cb43cf7576c6854c7a303ef3d7be866dc44de34" \
-    "dd668b609002b3578f2db83a1a684d706155bba2fc801cd411359fdd48218d00" \
-    "98f9876d14b39fc7ec863c6b1b73c41a7653f886901b3ead0c4ca9215a688408"
-
-
-$runner test_scala_version "2.12.3" \
-    "99d28c90ef1b8569da76a7e04415184cc54b57221ee259ffc55b2fcd64fb2107" \
-    "a8dd181a996dcc53a8c0bbb554bef7a1a9017ca09a377603167cf15444a85404" \
-    "93db412846912a1c212dd83c36dd51aa0adb9f39bfa6c4c3d65682afc94366c4"
-
-
-$runner test_scala_version "2.12.4" \
-    "8b681302aac584f7234547eed04d2beeeb4a4f00032220e29d40943be6906a01" \
-    "17824fcee4d3f46cfaa4da84ebad4f58496426c2b9bc9e341f812ab23a667d5d" \
-    "ea70fe0e550e24d23fc52a18963b2be9c3b24283f4cb18b98327eb72746567cc"
-
-
-$runner test_scala_version "2.12.5" \
-    "a113394b6f857e69ef5d95b77114ec3f12cb0e14d9ede32de0bbc9c36d677455" \
-    "c2636320d6479c82f2da6b8d76a820de9345a61327e648d4298a0048154fb87c" \
-    "27036d7574afff72294f0e63d54aa13acd8b16b525d51475691118b835e626e7"
-
-
-$runner test_scala_version "2.12.6" \
-    "3023b07cc02f2b0217b2c04f8e636b396130b3a8544a8dfad498a19c3e57a863" \
-    "f81d7144f0ce1b8123335b72ba39003c4be2870767aca15dd0888ba3dab65e98" \
-    "ffa70d522fc9f9deec14358aa674e6dd75c9dfa39d4668ef15bb52f002ce99fa"
-
-
-$runner test_scala_version "2.12.7" \
-    "6e80ef4493127214d31631287a6789170bf6c9a771d6094acd8dc785e8970270" \
-    "8f3dc6091db688464ad8b1ee6c7343d7aa5940d474ee8b90406c71e45dd74fc0" \
-    "7427d7ee5771e8c36c1db5a09368fa3078f6eceb77d7c797a322a088c5dddb76"
-
-
-$runner test_scala_version "2.12.8" \
-    "f34e9119f45abd41e85b9e121ba19dd9288b3b4af7f7047e86dc70236708d170" \
-    "321fb55685635c931eba4bc0d7668349da3f2c09aee2de93a70566066ff25c28" \
-    "4d6405395c4599ce04cea08ba082339e3e42135de9aae2923c9f5367e957315a"
-
-
-$runner test_scala_version "2.12.9" \
-    "5fd556459fd189b820db7d7c0a644ea5f7e8e032c421f2ad47038e72247fbf65" \
-    "364ee6ffd45f4fb8f9de40d1473d266ed5c199a44c1d4e2bdc895b1fbe35c75f" \
-    "4285ba64044d1a62b19304fe3ddd0088da240649c9fe2a6571c989feda1d0829"
-
-
-# Intermediate versions of 2.11.x
-
-
-$runner test_scala_version "2.11.10" \
-    "b70b748857213efe6f3a47d66acfa014c1bf51af3178b3a946eaae09f709fecc" \
-    "14a520328ea4ca7f423b30154a54d3df0a531a9c51f5e98eda272c9821bc5331" \
-    "fd896db4806875f538843ea24411e483ee4d0734710a108d0308ef108e83cf80"
+$runner test_scala_version "2.13.6"
+$runner test_scala_version "2.13.7"
```

Comment:
Overrides are still needed for versions not defined in default Rules Scala repositories. `scala_config` only selects repository by it's major version, so in case `2.13.7`, still `2.13.6` is used as defined in https://github.com/bazelbuild/rules_scala/blob/master/third_party/repositories/scala_2_13.bzl#L3. Unfortunately there is no good way to check the overridden version and report it to the user.